src/ex4/dist/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>PokeDoList</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <div class="container">
+        <div class="top">
+            <h1 class="title">PokeDoList</h1>
+            <p>undone/done</p>
+            <label class="switch" id="switch">
+                
+                <input type="checkbox" id="switch-input">
+                <span class="slider round"></span>
+            </label>
+            <select id="order-select" class="filter-select">
+                <option value="all">ALL LIST</option>
+                <option value="atoz">A-Z ORDER</option>
+                <option value="ztoa">Z-A ORDER</option>
+                <option value="dtou">DONE UNDONE ORDER</option>
+                <option value="utod">UNDONE DONE ORDER</option>
+            </select>
+        </div>
+        <div class="loader" id="loader"></div>
+        <div class="input-container">
+            <input class="todo-input" id="todo-input" type="text" placeholder="Add your new todo">
+            <button class="button add-button" id="add-todo-button"><i class="fas fa-plus"></i></button>
+        </div>
+        <div class="empty-todos-message" id="enter-todos"><h1 class="enter-todo-title">Enter Todos</h1></div>
```

Comment:
this causes some flickering on the initial load. consider adding it after the initial loading of todos (in case the list is empty)